internal/cli/cmd/dac/setup/setup.go

```diff
@@ -23,17 +23,23 @@ import (
 
 	persesCMD "github.com/perses/perses/internal/cli/cmd"
 	"github.com/perses/perses/internal/cli/config"
-	"github.com/perses/perses/internal/cli/cue"
 	"github.com/perses/perses/internal/cli/output"
-	"github.com/perses/perses/internal/cli/sources"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
 )
 
 const (
-	cueLanguage    = "cue"
-	goLanguage     = "go"
-	cueSchemasPath = "cue/"
+	cueLanguage      = "cue"
+	goLanguage       = "go"
+	cueSchemasPath   = "cue/"
+	minVersionForGo  = "v0.44.0" // Release that introduced the Go SDK
+	minVersionForCue = "v0.47.0" // TODO should be changed to v0.51.0 once released (Release that brought the move to CUE's new modules)
```

Comment:
I think you can already set `v0.51.0-beta.0`.

Because with the changes proposed, if I want to use cuelang, it won't be possible to use the v0.47.0 as it's not available from the registry

Reply:
Yes but this is actually to have the test passing, since v0.51.0-beta.0 doesnt exist yet :D So either we go with this "lie" for now, or we release a beta without this PR and it would be part of the next pre-release. WDYT?

Reply:
yeah maybe let's wait the release before merging this PR. That would avoid an issue if someone want to try it already

Reply:
should be good to update the version now